pytensor/tensor/shape.py

```diff
@@ -994,6 +995,9 @@ def specify_broadcastable(x, *axes):
     if not axes:
         return x
 
+    if min(axes) < 0:
+        axes = list(normalize_axis_tuple(axes, x.type.ndim))
```

Comment:
The list doesn't seem needed
```suggestion
        axes = normalize_axis_tuple(axes, x.type.ndim)
```